src/ripple/app/ledger/LedgerConsensus.h

```diff
@@ -56,16 +56,16 @@ class LedgerConsensus : public Traits
 
     virtual LgrID_t getLCL () = 0;
 
-    virtual void gotMap (TxSet_t const& map) = 0;
+    virtual void gotMap (Time_t const& now, TxSet_t const& map) = 0;
```

Comment:
This is now [here](https://github.com/ripple/rippled/pull/1941/files#diff-dab2766c14d0ef8e760dc5e353fa7b9dR298), but still uses a time point specified in the traits class so that the unit test version is free to use whatever clock it wants for network time.